packages/yoga/src/Button/web/Button.jsx

```diff
@@ -14,10 +16,22 @@ const Button = forwardRef(
       small,
       secondary,
       icon: Icon,
+      isLoading,
       ...props
     },
     ref,
   ) => {
+    const SpinnerContainer = styled.div`
+      ${() =>
+        isLoading &&
+        css`
+          position: absolute;
+          top: 50%;
+          left: 50%;
+          transform: translate(-50%, -50%);
+        `}
+    `;
```

Comment:
A component should not be created inside render. It's a bad practise. Here is why:

https://www.youtube.com/watch?app=desktop&v=2sAdzy90GtE&feature=youtu.be

Reply:
Done! Thanks for your review!! 🚀

Reply:
@matheushdsbr `    const SpinnerContainer = styled.div` is still inside the `Button` function component. It should be created outside

Reply:
Done! o/